packages/plugin-color/src/Color.ts

```diff
@@ -101,32 +108,33 @@ export class Color<T extends ColorConfig = ColorConfig> extends JWPlugin<T> {
 
             if (range.start.ancestor(hasColor)) {
                 // Set the color style cache to the default color.
-                const currentCache = inline.cache.style || {};
-                inline.cache.style = { ...currentCache, [this.styleName]: defaultColor };
+                if (!inline.cache.style) {
```

Comment:
Not without refactoring that cache (which should be done at one point). Mostly because of the way we set/reset it:
```ts
if (!this.cache.modifiers) {
    this.cache.modifiers = this.getCurrentModifiers(range);
}
```

Reply:
I don't understand how that makes it impossible to always set it.

Reply:
In my memory, the quote code above essentially recalculates the current modifiers unless there was a cache (like, we pressed the "bold" button for instance), in which case it will leave it alone. This is not ideal, for sure, and should be redone at one point... Maybe @Goaman, who made this, can enlighten us?